### PR TITLE
ERROR group without drained_at

### LINKED ISSUE #1863: CLB drained_at error recovery
Right now when something is not right when extracting draining time otter errors and tries to converge foreover. It should instead set `node.drained_at` to `None` and continue converging. If planner finds out that it needs this info and is not there, can ERROR the group. 

Note that this is slightly related to #1443 where that change should also set `node.drained_at=None` if keeps going back forever.

### PR DESCRIPTION
**Pull request by manishtomar**
Fixes #1863.

### CODE REVIEW

**Comment by manishtomar** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
@glyph review please

File: otter/convergence/model.py [hunk 1]
```diff
@@ -572,11 +573,20 @@ class CLB(object):
...
+    lb_id = attr.ib()
+    node_id = attr.ib()
+
+
 @implementer(ILBNode, IDrainable)
 @attributes([Attribute("node_id", instance_of=basestring),
              Attribute("description", instance_of=CLBDescription),
              Attribute("address", instance_of=basestring),
-             Attribute("drained_at", default_value=0.0, instance_of=float),
+             Attribute("drained_at", default_value=None),
```
Comment (glyph) [inline otter/convergence/model.py]: Active nothing! Adding `None` to `drained_at` means needing to check every single access point in the codebase.  Perhaps this should be a private `attr` with a public `@property` that raises `DrainingUnavailable`?

File: requirements/mimic.txt [hunk 2]
```diff
@@ -1 +1 @@
-git+https://github.com/rackerlabs/mimic.git@f317ee5d322aa7cb2b117e2984dfe8ef10558cd5
+git+https://github.com/rackerlabs/mimic.git@594dd5c8e80b670fa2d0c42f5eec9645e14aa54e
```
Comment (glyph) [inline requirements/mimic.txt]: Hrm.  Perhaps we're a bit behind on doing a new Mimic release?

File: otter/convergence/model.py [hunk 3]
```diff
@@ -521,6 +521,7 @@ def is_done_draining(now, timeout):  # pragma: no cover
 
         :return: Whether the node is done draining.
         :rtype: `bool`
+        :raises: ``DrainingUnavailable`` if draining info is not available
```
Comment (glyph) [inline otter/convergence/model.py]: Under what conditions is draining info not available?

File: otter/convergence/model.py (hunk 1 above)
Comment (manishtomar) [inline otter/convergence/model.py]: Good idea!

**Pull request merged**